rpcd/playbooks/roles/rpc_maas/tasks/package_install.yml

```diff
@@ -21,9 +21,29 @@
     cache_valid_time: 600
   with_items: maas_apt_packages
 
-- name: Install pip packages
+- name: Create venv dir for maas
+  file: 
+    path: "{{ maas_venv }}"
+    state: directory
+
+- name: Copy pip conf to venv
+  copy: src=/root/.pip/pip.conf dest="{{ maas_venv }}/pip.conf"
+
+- name: Update pip
+  pip:
+    name: pip
+    extra_args: '--upgrade'
+    virtualenv: "{{ maas_venv_enabled | bool | ternary(maas_venv | dirname, omit) }}"
+
+- name: Install pip packages to venv
   pip:
     name: "{{ item }}"
-    state: latest
+    state: present
     extra_args: "{{ pip_install_options | default('') }}"
+    virtualenv: "{{ maas_venv_enabled | bool | ternary(maas_venv | dirname, omit) }}"
+    virtualenv_site_packages: "{{ maas_venv_enabled | bool | ternary('no', omit) }}"
```

Comment:
You don't need line 44. The default for this option is "no"[1], so might as well get rid of it.
[1] http://docs.ansible.com/ansible/pip_module.html#options